functions/src/aqi-calculation/purple-air-response.ts

```diff
@@ -91,7 +90,7 @@ async function purpleAirToFirestore(): Promise<void> {
 
     // Get the existing data from Firestore for this sensor
     const sensorDocData = sensorDoc.data() ?? {};
-    const purpleAirId = getPurpleAirId(sensorDocData.purpleAirId);
+    const purpleAirId: number = sensorDocData.purpleAirId;
```

Comment:
In this PR I don't see a place where we check for numbers (so I assume it happened in a recent PR), what new code ensured that we don't need to do this check anymore?

Reply:
I manually checked all sensor docs in Firestore used numbers, so there's no longer a mix.